### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Alisakimhoang**
I just try to update exercises

### CODE REVIEW

File: Exercises-1-Basic/README.md [hunk 1]
```diff
@@ -1 +1,18 @@
-# Exercises 1 - Basic
\ No newline at end of file
+# Exercises 1 - Basic
```
Comment (yevgenylvovski) [inline Exercises-1-Basic/README.md:1]: You should commit the actual java classes with the solution.

Comment (Alisakimhoang) [reply Exercises-1-Basic/README.md:1]: commit the actual java classes with the solution??? I don't understand what are you talking about, can you please give me an example?

Comment (yevgenylvovski) [reply Exercises-1-Basic/README.md:1]: image

You need to commit the java file itself where you are writing java code in eclipse.

I should be able to pull your branch and run your solution locally.

Comment (Alisakimhoang) [reply Exercises-1-Basic/README.md:1]: Okay I will try to find how to do it

**Comment by Alisakimhoang** [comment]
Okay I will try

File: Exercises-1-Basic/README.md (hunk 1 above)
Comment (yevgenylvovski) [reply Exercises-1-Basic/README.md:1]: check out this example PR that I created:
https://github.com/yevgenylvovski/Java-Practice/pull/10

On the most right tab you can see the changed files.
Ignore the `.gitignore` file and check only the java file.
This is how you need to commit your solutions, as java files, so I can right away run it in Eclipse.

**Pull request closed without merging**